### PR TITLE
Update the strings for unsupported calls

### PR DESCRIPTION
**Pull request by bmarty**
## Content

Iterate on the strings used to renders legacy call in:
- notification
- room list last message
- timeline

## Motivation and context

Closes https://github.com/element-hq/element-x-android/issues/3853

## Screenshots / GIFs

https://github.com/user-attachments/assets/7515b42d-daa3-4be8-ad6c-4a525de9ff14

## Tests

- Alice uses Element X
- Bob is using Element Web
- Bob starts a **voice** call using Element Web
- Alice should see all the unsupported strings regarding the incoming call as per the video above.

## Tested devices

- [ ] Physical
- [x] Emulator
- OS version(s):

## Checklist

- [ ] Changes have been tested on an Android device or Android emulator with API 23
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/element-hq/element-x-android/blob/develop/CONTRIBUTING.md#accessibility
- [ ] Pull request is based on the develop branch
- [ ] Pull request title will be used in the release note, it clearly define what will change for the user
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [ ] You've made a self review of your PR

### CODE REVIEW

File: libraries/eventformatter/impl/src/main/kotlin/io/element/android/libraries/eventformatter/impl/DefaultTimelineEventFormatter.kt [hunk 1]
```diff
@@ -52,13 +52,11 @@ class DefaultTimelineEventFormatter @Inject constructor(
...
                 stateContentFormatter.format(content, senderDisambiguatedDisplayName, isOutgoing, RenderingMode.Timeline)
             }
-            is LegacyCallInviteContent -> {
-                sp.getString(CommonStrings.common_call_invite)
-            }
             is CallNotifyContent -> {
                 sp.getString(CommonStrings.common_call_started)
             }
             RedactedContent,
+            is LegacyCallInviteContent,
```
Comment (bmarty) [inline libraries/eventformatter/impl/src/main/kotlin/io/element/android/libraries/eventformatter/impl/DefaultTimelineEventFormatter.kt:59]: The case `is CallNotifyContent` should probably be moved here as well, but this is a bit out of scope of this PR.

**Comment by jmartinesp** [comment]
Just to confirm, is the text used for the timeline item the right one? The issue mentions using:

> Unsupported call. The new Element X app is needed to join this call.

But the included text seems to be different.

**Comment by bmarty** [comment]
> Just to confirm, is the text used for the timeline item the right one? The issue mentions using:
> 
> > Unsupported call. The new Element X app is needed to join this call.
> 
> But the included text seems to be different. Also, just having "Unsupported call" as the notification messages seems weird to me, there's plenty of space to explain why it's not supported.

This wording is in the section `Legacy side`. This PR is for Element X, so it would be strange to ask the user to use Element X in the Element X app (I do not blame you, I am also confused sometimes with all this legacy / x things)

About the notification content, I have done what's been done on iOS in https://github.com/element-hq/element-x-ios/pull/3502/files

**Review by jmartinesp — APPROVED** [review]
LGTM then, thanks.
